lib/RTCSession.js

```diff
@@ -3523,4 +3541,14 @@ module.exports = class RTCSession extends EventEmitter
       video
     });
   }
+
+  _getDTMFRTPSender()
+  {
+    const dtmfSender = this._connection.getSenders().find((sender) =>
+    {
+      return sender.track && sender.track.kind === 'audio';
+    });
+
+    return dtmfSender && dtmfSender.dtmf;
```

Comment:
Can we please avoid this `&&` usage and instead do:

```js
if (dtmfSender)
{
  return dtmfSender.dtmf;
}
```

?

Reply:
Done @ibc